contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.1.0--2.2.0.sql

```diff
@@ -2948,6 +2948,334 @@ $$
 LANGUAGE 'pltsql';
 GRANT EXECUTE on PROCEDURE sys.sp_helpuser TO PUBLIC;
 
+-- Need to add parameter for tsql_type_max_length_helper 
+ALTER FUNCTION sys.tsql_type_max_length_helper RENAME TO tsql_type_max_length_helper_2_1;
```

Comment:
Let's use `tsql_type_max_length_helper_deprecated_in_2_2`.

Reply:
Sure, I can do that